app/components/search/SearchResults/SearchResults.jsx

```diff
@@ -152,26 +152,18 @@ const SearchResult = ({ hit, index, setCenterCoords }) => {
     entryId = serviceId;
   }
 
-  // Todo: mock data until API returns handouts on the service model
-  const handoutCollection = [
-    { id: '1', description: 'English', link: 'https://ucsf.app.box.com/s/233qfwg6eilw1i1tipo9ts2jnh3mqz00' },
-    { id: '2', description: 'Cantonese', link: 'https://ucsf.app.box.com/s/233qfwg6eilw1i1tipo9ts2jnh3mqz00' },
-    { id: '3', description: 'Mandarin', link: 'https://ucsf.app.box.com/s/233qfwg6eilw1i1tipo9ts2jnh3mqz00' },
-    { id: '4', description: 'Filipino', link: 'https://ucsf.app.box.com/s/233qfwg6eilw1i1tipo9ts2jnh3mqz00' },
-    { id: '5', description: 'Spanish', link: 'https://ucsf.app.box.com/s/233qfwg6eilw1i1tipo9ts2jnh3mqz00' },
-  ];
-
   return (
     <div className={styles.searchResult}>
       <Texting closeModal={toggleTextingModal} service={service} isShowing={textingIsOpen} />
       <ClinicianActions
         isOpen={clinicianActionsIsOpen}
         setIsOpen={toggleClinicianActionsModal}
+        actions={(hit.instructions && hit.instructions[0]) ?? []}
```

Comment:
We might have assumed index 0 would be Clinician Actions and index 1 would be Covid Actions? Seems fairly brittle. Anyhow, I think we might want to just leave it like this until he and I meet, which may be tomorrow.

Reply:
Yeah, I agree that it's sketchy to assume that index 0 is the Clinician Actions and 1 is the Covid Actions, especially if we're not careful to sort the instructions on the API end before returning the results to the frontend.

Separately, I was checking out this branch locally and found it a little bit confusing as a user that we show these handouts and actions links even if there are no handouts or actions. I can't comment on the actual line of code where we show the links, since they're not in the diff, but I think we'll probably want to conditionalize them so that they don't display if there are no handouts or actions, respectively.

Reply:
Thanks for the feedback. @jjfreund may have a work session tonight and I  have made a note to bring it up. 

I was wondering this too. I thought alternatively that we may want to show a default "No results" message in the modal rather than hiding the button, but I'm not sure. I like the consistency of showing the buttons across all results. Perhaps we can disable it also. I have made a note to bring up in the design meeting to see what design thinks